holoviews/plotting/renderer.py

```diff
@@ -171,7 +177,10 @@ def get_plot(self_or_cls, obj, renderer=None):
                 initialize_dynamic(obj)
             obj = Compositor.map(obj, mode='data', backend=self_or_cls.backend)
 
-        if not renderer: renderer = self_or_cls.instance()
+        if not renderer:
+            renderer = self_or_cls
+            if inspect.isclass(self_or_cls):
```

Comment:
Is ``inspect`` really needed? Would something like ``if isinstance(self_or_cls, type) and issubclass(self_or_cls, Renderer)`` not work?

Reply:
Tbh, even ``not isinstance(self_or_cls, Renderer)`` should be enough.

Reply:
I prefer not to use ``inspect`` unless it is really necessary...